### PR TITLE
whitestar.cpp: Cleanup

### PR DESCRIPTION
**Pull request by clawgrip**
* Add display PLD [Victor Fernandez (City Game)].
* Add display version on game names (very useful when looking for undumped sets, but also informative to final users).
* Reformat comments.
* Add spacing, newlines, etc. to improve readability.

### CODE REVIEW

**Comment by galibert** [comment]
Conflicts happened, wanna take a look?

**Comment by clawgrip** [comment]
Done, thanks!!

**Comment by Quench0** [comment]
> * Add display PLD [Victor Fernandez (City Game)].

The PAL at U213 is an address decoder for the primary CPU on the mainboard. It's unrelated to the display controller.

I don't have access to this dump addition but the JED dump that's been floating around for many years has an equation error on pin 13. Where can I get this dump to verify?

**Comment by clawgrip** [comment]
Thanks, I've changed the comment for the PLD.
The dump was sent to The Dumping Union.

**Comment by Quench0** [comment]
> The dump was sent to The Dumping Union.

I don't have access to The Dumping Union.

There are three known versions of the U213 PAL which have different color dots. Those marked with a blue dot are the most common.
Sharkeys Shootout has a specific one with a gold dot. Lord of the Rings LE fitted with an optional shaker motor used a different version (for protection purposes) I don't have details on.
Terminator 3 lists the blue dot version in the manual.

Here is a version that includes proper pin 13 equation which should tri-state if you want to verify.
https://pinside.com/pinball/forum/topic/anyone-have-files-for-the-pal-chips-on-whitestar-boards#post-6712307

File: src/mame/pinball/whitestar.cpp [hunk 1]
```diff
@@ -6752,371 +9129,389 @@ ROM_START(sopranoi_107)
...
-GAME(2005,  sopranol_107, sopranol,  whitestarm,  whitestar, whitestar_state, empty_init,  ROT0,   "Stern",    "The Sopranos (1.07 Spain)",                     MACHINE_IS_SKELETON_MECHANICAL | MACHINE_SUPPORTS_SAVE )
-GAME(2005,  sopranoi,     0,         whitestarm,  whitestar, whitestar_state, empty_init,  ROT0,   "Stern",    "The Sopranos (5.00 Italy)",                     MACHINE_IS_SKELETON_MECHANICAL | MACHINE_SUPPORTS_SAVE )
-GAME(2005,  sopranoi_400, sopranoi,  whitestarm,  whitestar, whitestar_state, empty_init,  ROT0,   "Stern",    "The Sopranos (4.00 Italy)",                     MACHINE_IS_SKELETON_MECHANICAL | MACHINE_SUPPORTS_SAVE )
-GAME(2005,  sopranoi_300, sopranoi,  whitestarm,  whitestar, whitestar_state, empty_init,  ROT0,   "Stern",    "The Sopranos (3.00 Italy)",                     MACHINE_IS_SKELETON_MECHANICAL | MACHINE_SUPPORTS_SAVE )
-GAME(2005,  sopranoi_107, sopranoi,  whitestarm,  whitestar, whitestar_state, empty_init,  ROT0,   "Stern",    "The Sopranos (1.07 Italy)",                     MACHINE_IS_SKELETON_MECHANICAL | MACHINE_SUPPORTS_SAVE )
+/* Display ROM versions country/languaje codes:
```
Comment (galibert) [inline src/mame/pinball/whitestar.cpp:9155]: language :-)

Comment (clawgrip) [reply src/mame/pinball/whitestar.cpp]: Fixed, thanks!!!

**Pull request merged**